app/controllers/api/v4/converts_controller.rb

```diff
@@ -0,0 +1,35 @@
+class Api::V4::ConvertsController < Api::V4::ApplicationController
+
+  def create
+    params.require(:file)
+    params.require(:program)
+    if params[:file].respond_to?(:read)
+      file_text = params[:file].read
+      if file_text[8..29] == "org.fenix.llanfair.Run"
+        run_file = RunFile.new(file: RunFile.unpack_binary(file_text))
+      else
+        run_file = RunFile.new(file: file_text)
+      end
+    end
+
+    @run = Run.new(run_file: run_file, user: nil)
+    @run.convert(fast: params[:include_history] == "on" ? true : false)
+
+    extension = {'livesplit' => 'lss',
+      'urn' => 'json',
+      'splits_io' => 'json'
```

Comment:
Prefer just "json" instead of "splits_io", I don't think this format is official enough to give it a name